frontend/storage/models/Game.ts

```diff
@@ -0,0 +1,14 @@
+export interface GameProps{
```

Comment:
You could probably replace this with:

```
// If you plan to not require all fields defined in Game
export type GameProps = Pick<Game, 'gameId' | 'gameName'>;

// If you want to make them all optional
export type GameProps = Optional<Game>;

// If you want to ensure they all are provided
export type GameProps = Required<Game>;
```

[More info here.](https://www.typescriptlang.org/docs/handbook/utility-types.html)